insights/specs/default.py

```diff
@@ -638,6 +638,7 @@ class DefaultSpecs(Specs):
     testparm_v_s = simple_command("/usr/bin/testparm -v -s")
     thp_enabled = simple_file("/sys/kernel/mm/transparent_hugepage/enabled")
     thp_use_zero_page = simple_file("/sys/kernel/mm/transparent_hugepage/use_zero_page")
+    timedatectl = simple_command('/usr/bin/timedatectl')
```

Comment:
@huali027 could you use the command `timedatectl show --all` instead to produce machine readable form and simplify the parser?

Reply:
@huali027 this spec requires an entry in `insights/specs/insights_archive.py`.

Reply:
> @huali027 could you use the command `timedatectl show --all` instead to produce machine readable form and simplify the parser?
@bfahr `timedatectl show` is only supported on RHEL8 and later.